shell/install.js

```diff
@@ -17,6 +17,11 @@ ls("packages").forEach((loc) => {
     const pkg = require(pkgPath);
 
     cd(pkgDir);
+
+    if (pkgDir === "react-wildcat-handoff") {
+        exec(`npm run build`);
```

Comment:
This has to get built _before_ jspm linking happens, or else you'll receive a bunch of 404 errors on react-wildcat-handoff/dist/client.js